README.md

```diff
@@ -20,4 +20,7 @@ To get started working with this project locally you need to have node installed
 npm run dev
 ```
 
+## Documentation
+- [Discord.js](https://discord.js.org/#/docs/main/stable/general/welcome)
+- [dotenv](https://www.npmjs.com/package/dotenv)
```

Comment:
I don't think we need to link docs for dotenv. It should "just work" without us doing much besides setting up the 1 call (#2).

Reply:
Agreed and removed.